test.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
- [ ] This file (text.md) should not committed to Github kindly remove it and push the changes so it will be deleted from your repo, and it should be included in the `.gitignore` file

Reply:
Hi Ismailco. Thank you for your quick reply. I had included test.md in .gitignore file.
Perhaps I might missed how to include a file in .gitignore file. Thanks

Reply:
@amare1990, sorry for the late replay, you can just remove the test.md from your local repo and then push the changes to Github, when you create test.md the next time it will not be committed to github